storage/sealer/sched.go

```diff
@@ -139,6 +141,13 @@ type workerResponse struct {
 	err error
 }
 
+type rmRequest struct {
+	id     uuid.UUID
+	rmresE chan error
+	rmresC chan struct{}
```

Comment:
* Can be a single channel, `nil` error indicating success.
* The `rm` prefix is redundant, it's already in the struct name.
```suggestion
	res chan error
```